docs-source/index.md

```diff
@@ -174,37 +177,50 @@ In your controller:
 [HttpGet]
 public async Task<IActionResult> Get() {
   ...
-  var httpClient = services.Get<HttpClient>();
-  var management = new ManagementApiClient(token, baseUrl, httpClient);
+  var managementConnection = services.Get<IManagementConnection>();
+  var management = new ManagementApiClient("token", "YOUR_AUTH0_DOMAIN", managementConnection);
   ...
 }
 ```
 
-`ManagementApiClient` and `AuthenticationApiClient` can still be disposed using this pattern and will dispose of the `ApiConnection`. 
-The `ApiConnection` is smart enough to know not to dispose `HttpClient` instances it did not create and thus will share the `HttpClient` successfully.
+`ManagementApiClient` and `AuthenticationApiClient` also implement `IDisposable` and it is good practice to dispose of them when you are done.  
+
+When they have created their own `HttpClient` they will then dispose of it.  Any externally passed `HttpClient` will not be disposed as it is assumed to be shared with other instances.
+
+## Less-common scenarios
+
+### Mocking the ManagementApiClient
+
+While the `AuthenticationApiClient` has an associated `IAuthenticationApiClient` interface you will find that `ManagementApiClient` does not. The reason for this is that the such an interface would encompass the entire management API surface and that any slight tweak to the Management API - which happens very frequently - would break existing users of that interface.
```

Comment:
```suggestion
While the `AuthenticationApiClient` has an associated `IAuthenticationApiClient` interface you will find that `ManagementApiClient` does not. The reason for this is that the such an interface would encompass the entire management API surface and that any change to the Management API would break existing users of that interface.
```